uiServices/src/configServices.go

```diff
@@ -193,3 +199,19 @@ func putConfigs(rw http.ResponseWriter, req *http.Request) {
 
 	rw.WriteHeader(http.StatusNoContent)
 }
+
+func putConfigFileName(rw http.ResponseWriter, req *http.Request) {
+	path := getConfigHeader(req)
+	configFileName := chi.URLParam(req, "configFileName")
+	newConfigFileName := chi.URLParam(req, "newConfigFileName")
+
+	if len(newConfigFileName) < 1 {
+		logrus.Error("File path does not exist")
```

Comment:
log message is incorrect.
1. is it file path or file name?
2. message should say it is empty